CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Mattermost Mobile Apps Changelog
 
+## v1.13.0 Release
+- Release Date: October 16, 2018
+- Server Versions Supported: Server v5.4+ is required, Self-Signed SSL Certificates are not supported
```

Comment:
This is not true, the mobile app still supports server version 4.10+ which is the ESR but **IF** you have a 5.4 server the minimum app version is 1.13.

Example app version 1.12 and server 5.4 file upload won't work but app version 1.13 and server 4.10 works